resources/views/layouts/header_plain.blade.php

```diff
@@ -47,6 +47,9 @@
         <script>
             window.appInstance = "{{ env('APP_INSTANCE', 'base') }}";
             window.appDebug = "{{ env('APP_DEBUG', '0') }}";
+            window.Laravel = {
```

Comment:
This is intended as a single monolithic global variable, intended for frontend config? In the spirit of our monorepo JS `App` variable?

window.App or window.Global or window.Framework seems like a code smell. We can _at least_ segment by some namespace or feature, for modularity.

I recommend we avoid this approach. But that's not a blocker either.

Reply:
This would mainly be for the frontend config.

I agree this is not the best approach. We could probably use Vue's [Provide/Inject]( https://vuejs.org/guide/components/provide-inject) feature instead.

That said, I'm still getting familiar with the usage and setup of Vue with this Laravel app, so this might just be an interim solution that cleans up the current globals pattern.